backend/database/README.md

```diff
@@ -9,3 +9,11 @@ General information about Alembic migrations can be found [here](https://alembic
 3. Head over to the file that was just created. The path will be something like `backend/database/versions/xxxxxxxxxxxx_purpose_of_migration.py`. Edit the `upgrade()` and `downgrade()` functions such that `upgrade()` contains the commands to perform the migration you would like and `downgrade()` contains the commands to undo it.
 4. Once you've completed the changes (and at this point, you can create a PR to get the functions reviewed), test your migration. Ensure that you are [connected](https://github.com/chanzuckerberg/corpora-data-portal/blob/main/backend/chalice/api_server/README.md#development) to the database you'd like to migrate (usually `TEST`). Run `DEPLOYMENT_STAGE=test make db/migrate`.
 5. If no errors are outputted, complete the migration by running `DEPLOYMENT_STAGE=dev make db/migrate`. Ensure that tunneling is set up and `CORPORA_LOCAL_DEV` is set to true.
+
+### How to autogenerate migration script
+
+1. From the top level directory (`corpora-data-portal`), `cd backend`.
+1. Makes changes to ./corpora/common/corpora_orm.py
+1. Set DEPLOYMENT_STAGE=dev, CORPORA_LOCAL_DEV=1
+1. Run `make db/connect`
+1. Follow [How to perform a database migration](#how_to_perform_a_database_migration) instructions except use `make db/new_migration_auto MESSAGE="purpose_of_migration"`
```

Comment:
Can you add a note about double checking autogenerated migrations -- maybe call out common failures such as column renames?

Reply:
added a link to what alembic autogen does not detect

Reply:
thanks!